pkgs/applications/networking/irc/weechat/scripts/weechat-otr/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, substituteAll, buildEnv, fetchgit, fetchFromGitHub, pythonPackages, gmp }:
+
+let
+  # pure-python-otr requires an older version of pycrypto, which is not
+  # compatible with pycryptodome. Therefore, the latest patched version of
+  # pycrypto will be fetched from the Debian project.
+  # https://security-tracker.debian.org/tracker/source-package/python-crypto
+
+  pycrypto = pythonPackages.buildPythonPackage rec {
+    name = "pycrypto-${version}";
+    version = "2.6.1-9";
+
+    src = fetchgit {
+      url = "https://salsa.debian.org/sramacher/python-crypto.git";
+      rev = "debian/${version}";
+      sha256 = "0q8c5i1b7gbvsmsy28d32vk8vkxa2dqja5hzkb61kqqk309sj7x1";
+    };
+
+    buildInputs = [ gmp ];
+
+    preConfigure = ''
+      sed -i 's,/usr/include,/no-such-dir,' configure
+      sed -i "s!,'/usr/include/'!!" setup.py
+    '';
+  };
+
+  potr = pythonPackages.potr.overridePythonAttrs (oldAttrs: {
+    propagatedBuildInputs = [ pycrypto ];
+  });
+in stdenv.mkDerivation rec {
```

Comment:
bit nitpicky but might look nicer with a line break somewhere round here

Reply:
I don't think there is a style guide for this. Scrolling through nixpkgs' manual brought up another `in stdenv.mkDerivation` in the same line. The [`let` example](https://nixos.org/nix/manual/#sect-let-expressions) in nix's manual does also continues on the same line.